### DISCUSSION TITLE
OpenAPI requestBody description under schema instead of requestBody

### DISCUSSION THREAD

**Discussion by mavwolverine** [fastapi/fastapi#9876] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
async def update_pet(
    pet: Pet = Body(Required, description="Update an existent pet in the store"),
```

### Description

This renders as:
```yaml
      requestBody:
        content:
          application/json:
            schema:
              title: Pet
              allOf:
                - $ref: '#/components/schemas/Pet'
              description: Update an existent pet in the store
```

Instead it should render as:
```yaml
      requestBody:
        description: Update an existent pet in the store
        content:
          application/json:
            schema:
              title: Pet
              allOf:
                - $ref: '#/components/schemas/Pet'
```

Then it shows up in the Swagger UI.

https://github.com/tiangolo/fastapi/blob/master/fastapi/openapi/utils.py#L151
Adding 
```
    if field_info.description:
        request_body_oai["description"] = field_info.description
```
displays it in the UI

### Operating System

macOS

### FastAPI Version

0.95.1

### Python Version

3.11

**Comment by parekhs-deshaw** [comment]
@Kludex @tiangolo I'm also facing the same issue. How can we specify a description for `requestBody`?

**Comment by YuriiMotov** [comment, marked as answer]
You can add it using `openapi_extra` parameter of route decorator:

```
@app.post("/", openapi_extra={"requestBody": {"description": "my descr"}})
def update_pet(
    pet: Pet = Body(Required, description="Update an existent pet in the store"),
):
    pass
```

openapi:
```
      "post": {
        "summary": "Update Pet",
        "operationId": "update_pet__post",
        "requestBody": {
          "description": "my descr",
          "content": {
            "application/json": {
              "schema": {
                "allOf": [
                  {
                    "$ref": "#/components/schemas/Pet"
                  }
                ],
                "title": "Pet",
                "description": "Update an existent pet in the store"
              }
            }
          }
        },
```
